### ISSUE TITLE
`<b>` vs. `<strong>` etc...

### ISSUE THREAD

**Issue by aidantwoods** [jakwings/strictdown#3]
The reference parser produces and `<i>` and `<b>` tags when dealing with emphasis and strong emphasis respectively (spec uses the terms "Emphasis" and "Strong emphasis").

IMO the `<em>` and `<strong>` tags, respectively, should be used in place of these. It may seem like a pointless discrepancy, but I believe these tags to convey the intent of the spec more accurately (even though the spec uses `<i>` and `<b>` in its examples – maybe this could be adjusted too if this issue makes sense?).

`<b>` means "text should be bold" whereas `<strong>` means "the text should be strongly emphasised", the latter case may differ visually depending on context (e.g. strongly emphasised might be more appropriately contextually displayed in red, or a larger font, or underlined etc...). In the former case, the instruction "text should be bold" may not make it stand out in the context it is displayed in (simple example: all text is already bold due to default font).

Good summary here: https://developer.mozilla.org/en/docs/Web/HTML/Element/strong#Bold_vs._Strong

**Comment by iology** [comment]
Good points. [The comment](https://github.com/jakwings/strictdown/blob/210b02ecee9e862cbbd766baa988b54deede98ca/strictdown.js#L1563-L1566) in my code reminds me of why I chose `<i>/<b>` (because italic/bold could mean many things, "omnivorous"). So maybe I should say "italic/bold" instead of "emphasis/strong emphasis" in the document. :P

Reference:
http://accessibility.psu.edu/boldfacehtml/

**Comment by aidantwoods** [comment]
Which point exactly are you referencing in the link? My interpretation of it is that it is advocating em/strong over italic/bold :p

**Comment by iology** [comment]
Oh, I mean italic/bold can be reused for different purposes, like _emphasis_ (semantic use), **Terminology** (semantic), _Book Title_ (visual), _Just Italic_ (visual), _variable_ (visual).

If I choose `<em>, <strong>` only for semantic use, then I may bother with new syntax for `<b>, <i>`, or just mix them up.

**Comment by aidantwoods** [comment]
Ah I see what you mean – an instruction to explicitly be bold and not just "stand out" is sometimes necessary.

For those cases, would an author not also have to be in control of the way the text is displayed though (i.e. prevent the default font being bold or italic so that things that are explicitly so actually stand out). If they're in control of the way the text is displayed, then they could just opt to make `<em>` be displayed as italic, and `<strong>` be displayed as bold to achieve the same effect with nothing lost.

If the author controls the output method then they can choose how `<em>` and `<strong>` are styled, but if they do not then the output method *should* honour the intent for `<em>` and `<strong>` to be displayed differently to the surrounding text. But if the author controls the output method and uses `<i>` `<b>` then nothing has changed as far as the reader is concerned, whereas if they use `<i>` `<b>` and they do not control the output method then that text instruction may be indistinguishable from surrounding content – in this case do `<i>` and `<b>` not lose their value if they can't be distinguished by the reader?

To summarise that slightly differently: if the author controls the output, then they can pick particular stylings of `<em>`, `<strong>` to act like `<i>`, `<b>`; if they do not then `<i>` `<b>` might not be recognisable.

**Comment by iology** [comment]
You're right. So for a normal user, he does not see the difference. But an advanced user can customize the styling and explicitly use `<i>` and `<b>`.

**Comment by iology** [comment]
Now `<em>` and `<strong>` are used instead of `<i>` and `<b>`.

**Comment by aidantwoods** [comment]
Cheers, thanks for entertaining the idea :D
